lib/extension/listMPU.extension.js

```diff
@@ -1,9 +1,14 @@
 'use strict'; // eslint-disable-line strict
 
+const checkLimit = require('./tools').checkLimit;
+const DEFAULT_MAX_KEYS = 1000;
```

Comment:
I see that both Delimiter and ListMultipartUpload define a default of 1000, while the basic listing defines a default of 10000.

Shouldn't we ensure that the default is the same for all ? Is there a risk doing so ?